lib/advisor/core/advisory_finder.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Advisor.Core.AdvisoryFinder do
```

Comment:
I'm struggling to wrap my head around what the Advisories are. Are these the Advisors?